reference/architecture/authz_authn/authentication/index.md

```diff
@@ -214,7 +214,7 @@ flow looks like:
         IdentityProvider->>-Deck: HTTP 302 to https://gate.url/login?code=abcdef
     </div>
 
-1. OAuth provider confirms that the user is granting Gate access to his profile.
+1. OAuth provider confirms that the user is granting Gate access to their profile.
```

Comment:
I prefer "his or her," but Cloud Doc style is "their". So, yeah.

Reply:
Yeah, I read the Cloud Doc style guide for ideas on how to resolve this.